webservices_dispatch_action/version_updater.py

```diff
@@ -132,45 +121,38 @@ def update_version(git_repo, repo_name, input_version=None):
         )
     except Exception:
         LOGGER.exception("error while committing new recipe to repo")
-        return False, True
+        return False, True, new_version
 
-    return True, False
+    return True, False, new_version
 
 
-@click.command()
-@click.option(
-    "--feedstock-dir",
-    required=True,
-    type=str,
-    help="The directory of the feedstock",
-)
-@click.option(
-    "--repo-name",
-    required=True,
-    type=str,
-    help="The name of the repository",
-)
-@click.option(
-    "--input-version",
-    required=False,
-    type=str,
-    default=None,
-    help="The version to update to",
-)
-def main(
-    feedstock_dir,
-    repo_name,
-    input_version=None,
-):
-    git_repo = Repo(feedstock_dir)
-
-    _, version_error = update_version(
-        git_repo,
-        repo_name,
-        input_version=input_version,
-    )
-
-    if version_error:
-        sys.exit(1)
-    else:
-        sys.exit(0)
+def update_pr_title(
+    repo_name: str, pr_number: int, found_version: str
+) -> tuple[bool, bool]:
+    """
+    Returns [whether title changed, errored]
+    """
+    try:
+        with sensitive_env():
```

Comment:
You need to use the functions in the api utils to get the right token.

Reply:
This is still not right. The code uses a different token depending on which one is set. You should use the get actor token function and then feed that to function to make the api.

Reply:
Never mind. This looks good!